core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -193,11 +193,10 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </md-card>
       </div>
 
-      <div ng-if="myExplorationsView === 'card' && explorationsList.length > 0"
-           style="margin-top: 8px;">
-        <md-card ng-repeat="exploration in explorationsList track by exploration.id"
-                 class="oppia-activity-summary-tile oppia-dashboard-card-view-item"
-                 ng-click="showExplorationEditor(exploration.id)">
+      <div ng-if="myExplorationsView === 'card' && explorationsList.length > 0" style="margin-top: 8px;">
+        <md-card ng-repeat="exploration in explorationsList track by exploration.id" class="oppia-activity-summary-tile oppia-dashboard-card-view-item"
+                   ng-click="showExplorationEditor(exploration.id)">
+          <a ng-href="/create/<[exploration.id]>" style="text-decoration: none;">
```

Comment:
@rackstar17 `ng-href="/create/<[exploration.id]>"` and `ng-click="showExplorationEditor(exploration.id)"` are both doing the same thing. I would suggest that you must get rid of one of them!